pkg/services/hivelocity/device/device.go

```diff
@@ -502,15 +553,16 @@ func (s *Service) actionProvisionDevice(ctx context.Context) actionResult {
 		return actionError{err: errors.New(msg)}
 	}
 	if isPoweredOn {
-		msg := fmt.Sprintf("unexpected: device %d has power on.", deviceID)
-		record.Warnf(s.scope.HivelocityMachine, "ProvisionDeviceIsPoweredOn", msg)
-		return actionError{err: errors.New(msg)}
+		msg := fmt.Sprintf("device %d has power on. Waiting until it is shut-down.", deviceID)
+		record.Eventf(s.scope.HivelocityMachine, "ProvisionDeviceIsPoweredOn", msg)
+		return actionContinue{delay: 4 * time.Second}
 	}
 
 	userData, err := s.scope.GetRawBootstrapData(ctx)
 	if err != nil {
-		record.Warnf(s.scope.HivelocityMachine, "FailedGetBootstrapData", err.Error())
-		return actionError{err: fmt.Errorf("failed to get raw bootstrap data: %s", err)}
+		// This is common while starting a new cluster.
```

Comment:
fixed in 5b879e61

Reply:
I meant that in CAPH there are conditions for this.. not that we should return an error and let it fail...

Reply:
@janiskemper In caph it gets done like this:

```
	userData, err := s.scope.GetRawBootstrapData(context.TODO())
	if err != nil {
		return actionError{err: fmt.Errorf("failed to get user data: %w", err)}
	}
```
I think this change does not belong to the current PR. That's why I removed it. 

If this error comes again, and it makes sense to lower the log-level, then I will create a new PR. Is this ok for you?

Reply:
We have this function in HV to check whether bootstrap data is there: https://github.com/hivelocity/cluster-api-provider-hivelocity/blob/3c2839c3bd2e659bddf210b512cd3808c174e45e/pkg/services/hivelocity/device/device.go#L81

The original code here tried to add a second check for this AFAIK. Since this is not part anymore of this PR, we can resolve this.